packages/extras/src/lib/components/AnimatedSpriteMaterial/AnimatedSpriteMaterial.svelte

```diff
@@ -0,0 +1,269 @@
+<script lang='ts'>
+  import * as THREE from 'three'
+  import {
+    T,
+    asyncWritable,
+    type AsyncWritable,
+    createRawEventDispatcher,
+    useFrame,
+    watch,
+    useLoader,
+    useParent
+  } from '@threlte/core'
+  import type {
+    AnimatedSpriteProps,
+    AnimatedSpriteEvents,
+    AnimatedSpriteSlots,
+    SpriteJsonHashData,
+    FrameTag,
+    Frame
+  } from './AnimatedSpriteMaterial.svelte'
+  import { useTexture } from '../../hooks/useTexture'
+
+  type $$Props = Required<AnimatedSpriteProps>
+  type $$Events = AnimatedSpriteEvents
+  type $$Slots = AnimatedSpriteSlots
+
+  export let textureUrl: $$Props['textureUrl']
+  export let dataUrl: $$Props['dataUrl'] = ''
+  export let animation: $$Props['animation'] = ''
+  export let loop: $$Props['loop'] = true
+  export let autoplay: $$Props['autoplay'] = true
```

Comment:
I'm undecided on the autoplay default value... is it common practice to set it to `true` for sprite based stuff? Otherwise I'd set it to `false`

Reply:
If a change is happening, needs to be reflected in the docs as well